exercises/isogram/.meta/solutions/isogram.rb

```diff
@@ -1,9 +1,9 @@
 module BookKeeping
-  VERSION = 3
+  VERSION = 4
 end
 
 class Isogram
-  def self.is_isogram?(str)
+  def self.isogram?(str)
```

Comment:
Oh, you were commenting on an aspect of the example solution that has not been changed as part of this PR.

This PR is just trying to make the existing example pass the current tests, if you want to improve the example solution this can be done in another PR.